### PR TITLE
20.4.0 clear rn

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

### CODE REVIEW

File: Packs/AutoFocus/Playbooks/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
```
Comment (Itay4) [inline Packs/AutoFocus/Playbooks/CHANGELOG.md:4]: we have 20.4.0 twice here

File: Packs/Legacy/Integrations/GitHub/CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,10 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
```
Comment (Itay4) [inline Packs/Legacy/Integrations/GitHub/CHANGELOG.md:4]: we have 20.4.0 twice here

File: Packs/Legacy/Integrations/ImageOCR/CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
 -
```
Comment (Itay4) [inline Packs/Legacy/Integrations/ImageOCR/CHANGELOG.md:6]: remove one of the dashes

File: Packs/Legacy/Integrations/IntSight/CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
 -
```
Comment (Itay4) [inline Packs/Legacy/Integrations/IntSight/CHANGELOG.md:6]: remove one of the dashes

File: Packs/Legacy/Integrations/JoeSecurity/CHANGELOG.md [hunk 5]
```diff
@@ -1,4 +1,10 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
```
Comment (Itay4) [inline Packs/Legacy/Integrations/JoeSecurity/CHANGELOG.md:4]: we have 20.4.0 twice here

File: Packs/Legacy/Integrations/JsonWhoIs/CHANGELOG.md [hunk 6]
```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
 -
```
Comment (Itay4) [inline Packs/Legacy/Integrations/JsonWhoIs/CHANGELOG.md:6]: remove one of the dashes

File: Packs/Legacy/Integrations/MailSenderNew/CHANGELOG.md [hunk 7]
```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Integrations/MailSenderNew/CHANGELOG.md:5]: remove one of the dashes

File: Packs/Legacy/Integrations/MicrosoftTeams/CHANGELOG.md [hunk 8]
```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Integrations/MicrosoftTeams/CHANGELOG.md:5]: remove this dash

File: Packs/Legacy/Integrations/OnboardingIntegration/CHANGELOG.md [hunk 9]
```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Integrations/OnboardingIntegration/CHANGELOG.md:5]: remove one of the dashes

File: Packs/Legacy/Integrations/PaloAltoNetworks_Traps/CHANGELOG.md [hunk 10]
```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Integrations/PaloAltoNetworks_Traps/CHANGELOG.md:5]: remove one of the dashes

File: Packs/Legacy/Integrations/PolySwarm/CHANGELOG.md [hunk 11]
```diff
@@ -4,7 +4,6 @@ Added the *file* argument to the ***file*** command.
 ## [19.11.1] - 2019-11-19
 Improved handling of error messages.
```
Comment (Itay4) [inline Packs/Legacy/Integrations/PolySwarm/CHANGELOG.md:5]: this file was not cleaned

File: Packs/Legacy/Integrations/SMIME_Messaging/CHANGELOG.md [hunk 12]
```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Integrations/SMIME_Messaging/CHANGELOG.md:5]: remove one of the dashes

File: Packs/Legacy/Integrations/SlashNextPhishingIncidentResponse/CHANGELOG.md [hunk 13]
```diff
@@ -1,4 +1,10 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
```
Comment (Itay4) [inline Packs/Legacy/Integrations/SlashNextPhishingIncidentResponse/CHANGELOG.md:4]: we have 20.4.0 twice here

File: Packs/Legacy/Integrations/WindowsDefenderAdvancedThreatProtection/CHANGELOG.md [hunk 14]
```diff
@@ -1,4 +1,10 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
```
Comment (Itay4) [inline Packs/Legacy/Integrations/WindowsDefenderAdvancedThreatProtection/CHANGELOG.md:4]: we have 20.4.0 twice here

File: Packs/Legacy/Integrations/rasterize_old/CHANGELOG.md [hunk 15]
```diff
@@ -1,3 +1,6 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Integrations/rasterize_old/CHANGELOG.md:5]: remove one of the dashes

File: Packs/Legacy/Layouts/CHANGELOG.md [hunk 16]
```diff
@@ -0,0 +1,120 @@
+## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
```
Comment (Itay4) [inline Packs/Legacy/Layouts/CHANGELOG.md:4]: wow

File: Packs/Legacy/Playbooks/CHANGELOG.md [hunk 17]
```diff
@@ -0,0 +1,1002 @@
+## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
```
Comment (Itay4) [inline Packs/Legacy/Playbooks/CHANGELOG.md:4]: what is this?

File: Packs/Legacy/Scripts/CopyContextToField/CHANGELOG.md [hunk 18]
```diff
@@ -1,4 +1,10 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
```
Comment (Itay4) [inline Packs/Legacy/Scripts/CopyContextToField/CHANGELOG.md:4]: we have 20.4.0 twice here

File: Packs/Legacy/Scripts/DBotPredictOutOfTheBox/CHANGELOG.md [hunk 19]
```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Scripts/DBotPredictOutOfTheBox/CHANGELOG.md:5]: remove one of the dashes

File: Packs/Legacy/Scripts/DBotPredictPhishingLabel/CHANGELOG.md [hunk 20]
```diff
@@ -1,3 +1,6 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Scripts/DBotPredictPhishingLabel/CHANGELOG.md:5]: make the dash and text be in the same line

File: Packs/Legacy/Scripts/ExtractIndicatorsFromTextFile/CHANGELOG.md [hunk 21]
```diff
@@ -1,3 +1,6 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Scripts/ExtractIndicatorsFromTextFile/CHANGELOG.md:5]: remove one of the dashes

File: Packs/Legacy/Scripts/FindSimilarIncidentsByText/CHANGELOG.md [hunk 22]
```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Scripts/FindSimilarIncidentsByText/CHANGELOG.md:5]: make the dash and text be in the same line

File: Packs/Legacy/Scripts/FormattedDateToEpoch/CHANGELOG.md [hunk 23]
```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Scripts/FormattedDateToEpoch/CHANGELOG.md:5]: make the dash and the text be in the same line

File: Packs/Legacy/Scripts/GetFieldsByIncidentType/CHANGELOG.md [hunk 24]
```diff
@@ -1,3 +1,6 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Scripts/GetFieldsByIncidentType/CHANGELOG.md:5]: remove one of the dashes

File: Packs/Legacy/Scripts/GetIncidentsByQuery/CHANGELOG.md [hunk 25]
```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Scripts/GetIncidentsByQuery/CHANGELOG.md:5]: make the dash and the text be in the same line

File: Packs/Legacy/Scripts/HTMLtoMD/CHANGELOG.md [hunk 26]
```diff
@@ -1,3 +1,6 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Scripts/HTMLtoMD/CHANGELOG.md:5]: make the dash and the text be in the same line

File: Packs/Legacy/Scripts/NumberOfPhishingAttemptPerUser/CHANGELOG.md [hunk 27]
```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Scripts/NumberOfPhishingAttemptPerUser/CHANGELOG.md:5]: remove this dash

File: Packs/Legacy/Scripts/ParseEmailFiles/CHANGELOG.md [hunk 28]
```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Scripts/ParseEmailFiles/CHANGELOG.md:5]: make the dash and the text be in the same line

File: Packs/Legacy/Scripts/PcapHTTPExtractor/CHANGELOG.md [hunk 29]
```diff
@@ -1,3 +1,6 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Scripts/PcapHTTPExtractor/CHANGELOG.md:5]: remove this dash

File: Packs/Legacy/Scripts/SaneDocReport/CHANGELOG.md [hunk 30]
```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Scripts/SaneDocReport/CHANGELOG.md:5]: remove one of the dashes

File: Packs/Legacy/Scripts/StixParser/CHANGELOG.md [hunk 31]
```diff
@@ -1,4 +1,7 @@
 ## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
 -
```
Comment (Itay4) [inline Packs/Legacy/Scripts/StixParser/CHANGELOG.md:5]: make the dash and the text be in the same line

File: Packs/Legacy/Widgets/CHANGELOG.md [hunk 32]
```diff
@@ -0,0 +1,230 @@
+## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
```
Comment (Itay4) [inline Packs/Legacy/Widgets/CHANGELOG.md:4]: what

File: Packs/SafeBreach/Playbooks/CHANGELOG.md [hunk 33]
```diff
@@ -0,0 +1,9 @@
+## [Unreleased]
+
+
+## [20.4.0] - 2020-04-14
```
Comment (Itay4) [inline Packs/SafeBreach/Playbooks/CHANGELOG.md:4]: we have 20.4.0 twice here

File: Packs/Legacy/Widgets/CHANGELOG.md (hunk 32 above)
Comment (jochman) [reply Packs/Legacy/Widgets/CHANGELOG.md:4]: it's the release_notes_clear.py script...

File: Packs/Legacy/Integrations/PolySwarm/CHANGELOG.md (hunk 11 above)
Comment (jochman) [reply Packs/Legacy/Integrations/PolySwarm/CHANGELOG.md:5]: because it's merged with master.

**Pull request merged**